mysite/templates/psa/new_custom_login.html

```diff
@@ -0,0 +1,46 @@
+{% extends "ct/portal.html" %}
```

Comment:
@alexbojko our new templates should extend something new but not portal template.